include/seqan3/alignment/aligned_sequence/all.hpp

```diff
@@ -41,6 +41,7 @@
 
 #include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
 
-/*!\defgroup data_structure Data Structure
- * \brief Contains all data structures needed for the alignment module.
+/*!\defgroup aligned_sequnce Aligned Sequence
+ * \brief Contains the aligned_sequence_concept and custom adaptors which model
+ *        the concept and providing efficient access or modification.
```

Comment:
Gramma.

`\brief Provides seqan3::aligned_sequence_concept, as well as various ranges that model it.`